plugins/mentions/plugin.js

```diff
@@ -7,7 +7,8 @@
 
 ( function() {
 	var MARKER = '@',
-		MIN_CHARS = 2;
+		MIN_CHARS = 2,
+		cache = {};
```

Comment:
I think it's a good idea but not sure if it should be implemented as a simple object. If in the future we could like to extend mentions cache and we will end up with breaking API.

IMO, in that case, it should be implemented as `MentionsCache` object ([`Cache`](https://developer.mozilla.org/en-US/docs/Web/API/Cache) is already reserved). Currently it could just expose `set` and `get` functions.